segmenters/nlp/JiebaSegmenter/__init__.py

```diff
@@ -14,20 +14,43 @@ class JiebaSegmenter(BaseSegmenter):
      - all
 
     :type mode: str
+    :param pool_size: Number of parallel processes to use.
```

Comment:
Hello @jacobowitz ,

Thank you very much for the proposal. I would however not expose the multiprocessing at this level. Tipically parallelization in Jina is handled at Pod level by having multiple Peas inside a single Pod.

This could help at improving the throughput of a single request, but it may not play well with the current parallelization model in some cases. So at least parallelization should be disabled by default

Reply:
It is disabled by default as `pool_size` is set to one. But I was not aware of the Pod level handling of parallelization, which sounds indeed like better idea. Setting this to some high number here could potentially mess up resource management there I guess.
So let me remove the pool_size option quickly, will also address your other comment then and my failing test :)